src/functions/discord/client/$clientId.ts

```diff
@@ -0,0 +1,29 @@
+import { FunctionData } from "../../../typings/interfaces.js";
+import { escapeResult } from "../../../util/transpilerHelpers.js";
+
+export const $clientId: FunctionData = {
+    name: "$clientId",
+    type: "getter",
+    brackets: false,
+    optional: false,
+    fields: [],
+    version: "7.0.0",
+    default: [],
+    returns: "number",
+    description: "Returns the ID of client",
+    code: (data, scope) => {
+        // Getting the current scope
+        const currentScope = scope[scope.length - 1];
+
+        // Getting the client ID
+        const clientId = "__$DISCORD_DATA$__.client?.id";
```

Comment:
client user data is in client.readyData.user
aoiLuna can also do client.user

so u can either do client.readyData.user.id
or client.user.id